@types/phaser3-rex-plugins/plugins/gameobjects/dom/InputText.d.ts

```diff
@@ -0,0 +1,58 @@
+declare module 'phaser3-rex-plugins/plugins/gameobjects/dom/InputText' {
+    export interface TextChangedCallback {
+        (obj: Phaser.GameObjects.GameObject, text: string): void;
+    }
+
+    export default class InputText extends Phaser.GameObjects.DOMElement {
```

Comment:
https://github.com/rexrainbow/phaser3-rex-notes/blob/master/plugins/gameobjects/dom/inputtext/InputText.js#L223

Same suggestion `implements methods {` like in a `templates/ui/ui-plugin`

`Resize` returns `this`, and we can't just write `typeof Resize`, so I'm suggest to create type `ResizeType` 

Something analogical I did here:

https://github.com/khasanovbi/phaser3-rex-plugins-types/blob/master/%40types/phaser3-rex-plugins/plugins/board/board/chess/RemoveAllChess.d.ts

https://github.com/khasanovbi/phaser3-rex-plugins-types/blob/master/%40types/phaser3-rex-plugins/plugins/board/board/LogicMethods.d.ts#L136